modules/logging-feature-reference-5.6.adoc

```diff
@@ -0,0 +1,102 @@
+// Module is included in the following assemblies:
+//
+:_content-type: REFERENCE
+[id="logging-5-6-collector-ref_{context}"]
+
+= Collector features
+
+include::snippets/logging-outputs-5.6-snip.adoc[]
+
+.Log Sources
+[options="header"]
+|===============================================================
+| Feature                   | Fluentd  | Vector
+| App container logs        | &#10003; | &#10003;
+| App-specific routing      | &#10003; | &#10003;
+| App-specific routing by namespace | &#10003; | &#10003;
+| Infra container logs      | &#10003; | &#10003;
+| Infra journal logs        | &#10003; | &#10003;
+| Kube API audit logs       | &#10003; | &#10003;
+| OpenShift API audit logs  | &#10003; | &#10003;
+| Open Virtual Network (OVN) audit logs| &#10003; | &#10003;
+|===============================================================
+
+.Authorization and Authentication
+[options="header"]
+|=================================================================
+| Feature                     | Fluentd  | Vector
+| Elasticsearch certificates  | &#10003; | &#10003;
+| Elasticsearch username / password | &#10003; | &#10003;
+| Cloudwatch keys             | &#10003; | &#10003;
+| Cloudwatch STS              | &#10003; |
```

Comment:
I think STS is supported on both fluentd/vector.
cc. @IshwarKanse

Reply:
For Cloudwatch, yes.